src/commands/buildJs.ts

```diff
@@ -150,6 +150,16 @@ async function waitAllAndThrowIfAnyCompilationsFailed(
     throw new BuildJsCompilationError(reasons, results.length);
   }
   return reasons;
+
+  function parseErrorReason(
+    reason: CompilerError & { info?: Record<string, any> },
+    config: DuckConfig
+  ): { command: string; items: CompileErrorItem[] } {
+    const { message: stderr, info } = reason;
+    const message = config.batch === "aws" && info ? info.message : stderr;
```

Comment:
This issue occurs not only with `--batch aws` but also with `--batch local`.

Reply:
@teppeis Thanks. I fixed this. https://github.com/cybozu/duck/pull/378/commits/b8e1583d81489ebaa3f0c51a13550f9f143ac4ab